action-server/tests/actions/test_test_navigation_form.py

```diff
@@ -87,7 +109,14 @@
         }
     ],
     "image_url": "some_url",
-    "subtitle": "appointment known clientele referral = true",
+    "subtitle": "appointment known clientele referral = true\n\n"
+    + "Mon: 8:30am-12pm, 1pm-5pm\n"
```

Comment:
Funny thing, you don't need the + here. But it's much clearer like this and less error prone (without it, someone could think you forgot a comma somewhere)